src/components/UserDetail/Loans.js

```diff
@@ -0,0 +1,117 @@
+import { get } from 'lodash';
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import {
+  FormattedMessage,
+} from 'react-intl';
+
+import {
+  stripesShape,
+} from '@folio/stripes/core';
+
+import {
+  Col,
+  KeyValue,
+  Row,
+} from '@folio/stripes/components';
+
+import css from './UserDetail.css';
+
+function Loans({
+  resources,
+  stripes,
+  user,
+}) {
+  const renderOpenRequests = useMemo(() => {
+    if (!stripes.hasPerm('ui-users.requests.all,ui-requests.all')) return '-';
```

Comment:
should it be `<NoValue />` ?

Reply:
also it's a comma in the perm name

Reply:
@aliaksei-chumakou  it comes from https://github.com/folio-org/ui-checkout/pull/559/files/73103eb463aab60d75a2eb6ab9669d92b540d22a#diff-f144dfccb034512387301574922d335a1f1d763c081e1dfccd59e209ff0ef268L151

Reply:
It's a list of permissions, intentionally, I think.
It's parsed here
https://github.com/folio-org/stripes-core/blob/ad9f2cc9f5dd7b15ce624e948d6c5edfb9f9c109/src/Stripes.js#L102